pkg/storegateway/gateway_blocks_http.go

```diff
@@ -126,7 +124,7 @@ func (s *StoreGateway) BlocksHandler(w http.ResponseWriter, req *http.Request) {
 			DeletedTime:     formatTimeIfNotZero(deletedTimes[m.ULID].UTC(), time.RFC3339),
 			CompactionLevel: m.Compaction.Level,
 			BlockSize:       listblocks.GetFormattedBlockSize(m),
-			Labels:          lbls.WithoutLabels(tsdb.TenantIDExternalLabel).String(),
```

Comment:
My reasoning is to show all labels in the listing as sometimes it may be useful to know if block had any labels or not (even `__org_id__`). Previously ALL blocks had this label, so we didn't need to think about that. But now old blocks will have the label, while new ones will not, so we may need to check for that.

Reply:
Ok fair enough.